device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/proc/SimulatorWebDriverAgent.kt

```diff
@@ -25,7 +25,13 @@ class SimulatorWebDriverAgent(
     }
 
     private fun installHostApp() {
-        remote.fbsimctl.installApp(udid, File(hostApp))
+        val result = remote.execIgnoringErrors(listOf("xcrun", "simctl", "install", udid, hostApp))
+
+        if (!result.isSuccess) {
+            val errorMessage = "Failed to install WebDriverAgent $hostApp to simulator $udid. Result stdErr: ${result.stdErr}"
```

Comment:
Should we log complete results? I believe stdout might contain useful information for context.